rundeckapp/grails-app/services/rundeck/services/data/WebhookDataService.groovy

```diff
@@ -0,0 +1,12 @@
+package rundeck.services.data
+
+import grails.gorm.services.Service
+import webhooks.Webhook
+
+@Service(Webhook)
+interface WebhookDataService {
+
+    Webhook save(Webhook webhook)
+    void delete(Long id)
+
```

Comment:
lots of usage of `Webhook.<staticmethod>` in the GormWebhookDataProvider should be converted to use the WebhookDataService instead

Reply:
Given the recent issues we've had with the data services I think we should wait to change these usages until we have a more bug free experience. Let's leave the static method usages for now.